fedimint-core/src/encoding/btc.rs

```diff
@@ -66,10 +62,8 @@ impl crate::encoding::Decodable for bitcoin::psbt::Psbt {
         d: &mut D,
         _modules: &ModuleDecoderRegistry,
     ) -> Result<Self, crate::encoding::DecodeError> {
-        Ok(bitcoin29_to_bitcoin32_psbt(
-            &bitcoin29::consensus::Decodable::consensus_decode_from_finite_reader(d)
-                .map_err(crate::encoding::DecodeError::from_err)?,
-        ))
+        Self::deserialize_from_reader(&mut std::io::BufReader::new(d))
```

Comment:
You need to use the same simple reader impl you used everywhere else. The `BufReader` will throw away leftover bytes.

Reply:
Fixed